src/api/app/views/webui2/webui/kiwi/images/_package_fields.html.haml

```diff
@@ -9,38 +9,40 @@
       = link_to_remove_association f do
         %i.fas.fa-times.text-danger
 
-%div{ 'tabindex': '-1', role: 'dialog', id: "#{f.object.name.present? ? 'package-' + f.object.name : 'add-package'}",
-class: "modal fade #{'new_element' if f.object.new_record?}" }
-  .modal-dialog{ role: 'document' }
-    .modal-content
-      .modal-header
-        %h5.modal-title #{f.object.name.present? ? 'Edit' : 'Add'} package
+  %div{ 'tabindex': '-1', role: 'dialog', id: "#{f.object.name.present? ? 'package-' + f.object.name : 'add-package'}",
+  class: "modal fade #{'new_element' if f.object.new_record?}" }
```

Comment:
The classes `modal` and `fade` can be applied directly.

Reply:
done